src/components/helpers/form-helpers/render-date-pickerV2.jsx

```diff
@@ -4,7 +4,7 @@ import {
   MuiPickersUtilsProvider,
   KeyboardDatePicker,
 } from "@material-ui/pickers";
-import DateFnsUtils from "@date-io/date-fns";
```

Comment:
It doesn't need to be changed. It must remain

Reply:
done